subprojects/shipkit/src/test/groovy/testutil/GradleVersionsDeterminer.groovy

```diff
@@ -24,7 +24,7 @@ trait GradleVersionsDeterminer {
             case null:
                 return [currentGradleVersion()]
             case QUICK_GRADLE_VERSIONS_VALUE:
-                return [currentGradleVersion(), "4.1", "3.5.1", "2.14.1"].unique()
+                return [currentGradleVersion(), "4.2.1", "4.0.2"].unique()
```

Comment:
Did you mean `4.5.1` instead of `4.2.1`?

Reply:
thx for pointing that out!
It looks like we do have a problem with gradle 4.5.1. I will create a separate ticket for this one.